Add Landing Page for Infrastructure Documentation (#3102)
alan-turing-institute/the-turing-way: book/website/community-handbook/infrastructure.md

(ch-infrastructure)=

> # Infrastructure

Reviewer: I really like this write-up - sets the scene very nicely.

> The Turing Way consists of many parts which need to work together to make sure the book can be build, looks correct and is available on the internet.

Reviewer 2: ```suggestion
The Turing Way consists of many parts which need to work together to make sure the book can be built, renders correctly and is available on the internet.
```
Just some small suggestions for clarity & spelling!!

There are processes to build, deploy and ensure the quality of the book.
There is also configuration which may direct those processes or the site.

> Collectively we call these things infrastructure.

Reviewer 2: ```suggestion
While infrastructure can be defined in many ways, collectively we call these things infrastructure.
```
Just a suggestion for wording here - as we also use this definition to acknowledge its social elements.

> Some examples include

Reviewer 2: ```suggestion
Some examples include:
```

Reviewer 2: I'm wondering if some of these terms should be listed in the TTW Glossary, as folks may or may not be familiar with them? (Especially thinking of first-time users of Github for example, who might be contributing to TTW, or non-technical folks).

Author: Done in [62f6172](https://github.com/alan-turing-institute/the-turing-way/pull/3102/commits/62f617230069d5490029e9323a297ea12602f429)

> - Instructions to build the site from the source files

Reviewer 2: This could link directly to this page? https://the-turing-way.netlify.app/community-handbook/local-build.html

Author: That's interesting. When I wrote this I was thinking about CI/CD specifically.

We could mention both here?

Author: Done in [62f6172](https://github.com/alan-turing-institute/the-turing-way/pull/3102/commits/62f617230069d5490029e9323a297ea12602f429)
